apis/nucleus/src/components/listbox/ListBoxRowColumn.jsx

```diff
@@ -147,7 +147,7 @@ export default function RowColumn({ index, style, data, column = false }) {
       }
     }
     setClassArr(clazzArr);
-  }, [cell && cell.qState]);
+  }, [cell && cell.qState, checkboxes]);
```

Comment:
Otherwise we won't refresh the selection state when switching between checkboxes: true/false when using the switch. We will get checkbox fields with a green background e.g.

Reply:
the checkbox option will need to be split (maybe not in this PR) as for analytic creation we want checkboxes and showing the state

Reply:
Interesting, good to know 😄. Yes, not in the scope of this PR I think.